test/python/visualization/test_dag_drawer.py

```diff
@@ -84,34 +84,6 @@ def test_dag_drawer_no_register(self):
             image = Image.open(tmp_path)
             self.assertImagesAreEqual(image, image_ref, 0.1)
 
-    @unittest.skipUnless(_optionals.HAS_GRAPHVIZ, "Graphviz not installed")
-    @unittest.skipUnless(_optionals.HAS_PIL, "PIL not installed")
-    def test_dag_drawer_with_dag_dep(self):
-        """Test dag dependency visualization."""
-        from PIL import Image  # pylint: disable=import-error
-
-        bits = [Qubit(), Clbit()]
-        qr = QuantumRegister(4, "qr")
-        cr = ClassicalRegister(4, "cr")
-        qc = QuantumCircuit(qr, bits, cr)
-        qc.h(0)
-        qc.cx(0, 1)
-        qc.cx(0, 2)
-        qc.cx(0, 3)
-        with self.assertWarns(DeprecationWarning):
-            qc.x(3).c_if(cr[1], 1)
-        qc.h(3)
-        qc.x(4)
-        qc.barrier(0, 1)
-        qc.measure(0, 0)
-        dag = circuit_to_dagdependency(qc)
-        with tempfile.TemporaryDirectory() as tmpdirname:
-            tmp_path = os.path.join(tmpdirname, "dag_d.png")
-            dag_drawer(dag, filename=tmp_path)
-            image_ref = path_to_diagram_reference("dag_dep.png")
-            image = Image.open(tmp_path)
-            self.assertImagesAreEqual(image, image_ref, 0.1)
-
```

Comment:
This test should stay, just the `c_if` needs to go - we'd be losing the test coverage.

Reply:
Restored in: https://github.com/Qiskit/qiskit/pull/13506/commits/720dbe8246fff5e1b6cacb0f8016003a60dcfde6